doc/rst/technotes/gpu.rst

```diff
@@ -5,22 +5,24 @@
 GPU Programming
 ===============
 
-Chapel includes preliminary work to target NVIDIA and AMD GPUs.  This work is
-under active development and has not yet been tested under a wide variety of
-environments although we have tested it with NVIDIA Tesla P100, V100, and RTX
-A2000 GPUs; for AMD we have tested it with MI60 and MI100 GPUs.
+Chapel can be used to program GPUs. Currently  NVIDIA and AMD GPUs are
```

Comment:
Do you want to tease that we're planning to add future support for Intel here as well or do you think it's best to leave it out for now?